src/routes/sync.ts

```diff
@@ -0,0 +1,22 @@
+import { FastifyInstance } from 'fastify'
+import { syncProducts, syncPrices } from '../lib/sync'
+
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+export default async function routes(server: FastifyInstance) {
+  /**
+   * POST /products/sync
+   *    Sync products from Stripe
+   */
+  server.get('/sync', {
+    handler: async (request, reply) => {
+      const products = await syncProducts()
+      const prices = await syncPrices()
+      return reply.send({
+        statusCode: 200,
+        ts: Date.now(),
+        products,
+        prices,
+      })
+    },
+  })
+}
```

Comment:
we should probably store the status of this in a table and only allow one sync to be run at any given time